src/anomalib/loggers/mlflow.py

```diff
@@ -0,0 +1,104 @@
+"""MLFlow logger with add image interface."""
+
+from typing import Literal
+
+import numpy as np
+from matplotlib.figure import Figure
+
+# Maybe use try_import from anomalib.utils.exceptions instead.
+try:
+    from lightning.pytorch.loggers.mlflow import MLFlowLogger
+except ModuleNotFoundError:
+    print("To use mlflow logger install it using `pip install mlflow`")
```

Comment:
I agree, either `try_import` or `logging.error()`

Reply:
I changed the code to check if mlflow is installed using try_import. try_import should log a warning if mlflow is not installed.